.github/workflows/ci.yml

```diff
@@ -363,16 +363,12 @@ jobs:
     continue-on-error: true
 
     steps:
-      - name: Checkout snuba
-        uses: actions/checkout@v2
-
-      - name: Pull snuba CI images
-        run: |
-          docker pull ghcr.io/getsentry/snuba-ci:${{ github.sha }} || docker pull ghcr.io/getsentry/snuba-ci:latest
+      - name: Checkout Snuba
+        uses: actions/checkout@v3
       - name: Run Sentry self-hosted e2e CI
-        uses: getsentry/action-self-hosted-e2e-tests@5bbd3d6725bbe4bf5cc12f8a3291ce87df6a891f
+        uses: getsentry/action-self-hosted-e2e-tests@697a3ca95a72f93e3a1ebdb8f967be38d1f029f6
         with:
           project_name: snuba
-          local_image: ghcr.io/getsentry/snuba-ci:${{ github.sha }}
-          docker_repo: ghcr.io/getsentry/snuba
-          # TODO: add docker tag to build for
+          docker_repo: getsentry/snuba
+          image_url: us.gcr.io/sentryio/snuba:${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
Just curious, how come this changed from `github.sha` to `github.event.pull_request.head.sha`

Reply:
Ah, in PRs the Github SHA points to the merge commit, which isn't what we want. Not super important here but for consistency's sake it seems like a good move.